vllm/worker/model_runner.py

```diff
@@ -62,7 +62,7 @@
 # Capture graphs for token size 1, 2, 4, 8, 16, 24, 32, 40, ..., 256.
```

Comment:
also change this line please, 256 --> 1024

Reply:
Done.